src/benchmarks/Makefile

```diff
@@ -0,0 +1,15 @@
+CXX=g++
+HDF5_ROOT=/usr/local
```

Comment:
This is highly system dependent. Also all user-modifiable variables here should be set with:
>  There is another assignment operator for variables, `?='. This is called a conditional variable assignment operator, because it only has an effect if the variable is not yet defined. This statement: 
(see https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_6.html)

I think this would allow to grab these variables from the environment easier.